src/i18n.ts

```diff
@@ -122,9 +126,9 @@ export const safeTranslate = (translator: any, key: string | undefined) => {
     return translator(key);
   }
   // eslint-disable-next-line no-console
-  console.info(
-    `Translation for "${key}" is not configured in your translation file.`,
-  );
+  // console.info(
```

Comment:
I think this is useful to keep for now while we are "developin"

Reply:
In fact, this looks like a great log to keep permanently so future features are reminded to add translations for their feature. Possibly use the notification context to publish a visible popup if a translation is not found.

Reply:
yes I can put this log back, however it's just make the console window full of console.log() statements which actually interfere when i was debugging other issues

Reply:
hmm it should not be full of logs, otherwise it means one of:
1. the translation is not available
2. we are translating the same content multiple times